web/src/components/IdeEditor/IdeEditor.js

```diff
@@ -34,6 +35,7 @@ const IdeEditor = () => {
     if (key === 's' && (ctrlKey || metaKey)) {
       event.preventDefault()
       dispatch({ type: 'render', payload: { code: state.code } })
+      localStorage.setItem(codeStorageKey, state.code)
```

Comment:
I figured renders was a good time to "save" the code. I'm not sure what will happen to this local storage long term when the ability to save openscad projects is there, but its likely to stick around, because loosing work sucks.